product_barcode_generator/models/product.py

```diff
@@ -36,21 +36,20 @@ class ProductCategory(models.Model):
 class ProductProduct(models.Model):
     _inherit = 'product.product'
 
-    ean13 = fields.Char(copy=False)
     ean_sequence_id = fields.Many2one('ir.sequence', string='Ean sequence')
```

Comment:
What's the meaning of the sequence here? Shouldn't be enough at company level (although I miss the possibility of putting more than one sequence there)?